packages/esbuild/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "mocha \"dist/test/**/*.spec.js\""
   },
   "peerDependencies": {
-    "esbuild": "^0.17.18"
+    "esbuild": ">=0.17.18"
```

Comment:
This change declares a future 1 version as compatible.
I would say to limit it to `>=0.17.18 <1`, but that is the same as it was before (right?)

@AviVahl - is there a difference?

Reply:
@barak007 - esbuild itself is in version 0 and breaking changes are allowed in minor anyhow. so I think it's fine have no upper limit. When `esbuild@1` is out we can release a version with an upper limit. - so this is fine